### DISCUSSION TITLE
"left to spend"-box disappearing

### DISCUSSION THREAD

**Discussion by Studio5086** [firefly-iii/firefly-iii#6321] [Firefly III]
Maybe I miss a crucial point to this functionality, but upon reloading the dashboard, I can see a box with the text "left to spend" which eventually changes to "spent". In the /budgets tab, the "Left (per day) column is still visible. Am I missing something?

**Comment by JC5** [comment]
On the budgets page, set the "available amount".

**Reply by Studio5086** [reply]
Okay, wow. I somehow did not see that button at all... It works now. Does this field fill in automatically next month, when I've set my auto budget to "Set a fixed amount every period"?

**Reply by JC5** [reply]
No, but I want to change it into something more user friendly in future releases.

**Reply by Studio5086** [reply]
It would also be great to group revenue accounts/ income categories automatically as a budget each month (eg. fixed salary + dynamic freelancing revenue)

**Reply by tmllull** [reply]
> No, but I want to change it into something more user friendly in future releases.

Waiting for this. I think can be a good point to set this “general” budget automatically instead of to set manually every month, for example taking the total amount of the other budgets or with specific value. Or disable if the user can't use it.

My salary not changes a lot month by month, and if I “split” in different categories (and set a budget amount for every one), every month my “available amount” will be the total of these categories. If I get €1000/month, and I budget €900 in different categories, my monthly budget will be €900 every month.

Is just an idea :)
